docs/services.md

```diff
@@ -11,8 +11,20 @@ aliases:
 
 Services are built-in software packages that make it easier to add complex capabilities such as motion planning or object detection to your machine.
 
+{{< imgproc src="/viam/machine-components.png" alt="Machine components" resize="1000x" style="max-width:650px" >}}
```

Comment:
I think centering these might look better. I'm a bit concerned that it's not clear here what services are. Maybe the previous text should reference the services in this image

Reply:
Ok! I've requested a "SERVICES" label inside the services box for this and the other diagram. I'm also adding additional copy above here--it might be overkill though; I fear it makes this page too verbose. lmk what you think.